### PR TITLE
T10798 change switcher design

### PR DESCRIPTION
**Pull request by wagdy-git**
@amrelbialy 
@NermeenMoustafa

### CODE REVIEW

**Comment by NermeenMoustafa** [comment]
reviewed

File: packages/ui/src/core/switcher/switcher.mixin.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { WithTheme } from '@scaleflex/ui/theme/entity';
+import { FontVariant as FV } from '@scaleflex/ui/utils/types/typography';
+
+import { css } from 'styled-components';
```
Comment (amrelbialy) [inline packages/ui/src/core/switcher/switcher.mixin.ts:4]: Styled components should be at first line and we don't need space

File: packages/ui/src/theme/roots/palette/entity/default-palette.ts [hunk 2]
```diff
@@ -55,13 +55,20 @@ export const lightPalette = {
...
   [Color.BackgroundPrimaryActive]: 'rgba(93,109,126,1)',
   [Color.BackgroundPrimary_0_5_Opacity]: 'rgba(255,255,255,0.500)',
   [Color.BackgroundSecondary]: 'rgba(255,255,255,1)',
 
+  [Color.BackgroundSwitcherOff]: '#D8E0E8',
+  [Color.BackgroundSwitcherOn]: 'rgb(104,121,235)',
+  [Color.BackgroundInnerSwitcherOff]: '#A9B6C2',
+  [Color.BackgroundInnerSwitcherDisabled]: '#F3F7FA;',
+  [Color.BackgroundInnerSwitcherOn]: 'rgba(104, 121, 235, 0.4);',
+
```
Comment (amrelbialy) [inline packages/ui/src/theme/roots/palette/entity/default-palette.ts:71]: Check all the colors again

File: packages/ui/src/theme/entity/default-theme.ts [hunk 3]
```diff
@@ -28,8 +28,8 @@ const labelFontVariants = {
   },
   [FontVariant.LabelNormal]: {
     ...defaultFontValue,
-    fontSize: '14px',
-    lineHeight: '16px',
+    fontSize: '16px',
+    lineHeight: '18px',
```
Comment (amrelbialy) [inline packages/ui/src/theme/entity/default-theme.ts:32]: Check with figma

**Pull request merged**